### DISCUSSION TITLE
Will not connect

### DISCUSSION THREAD

**Discussion by geos1200** [WillCodeForCats/solaredge-modbus-multi#576] [General]
### Describe the bug

Newly installed 
2  SE7600H-US
2 LG RESU 16
Modbus integration will not connect
I have confirmed that Modbus over TCP is enabled on the leader
Devices are connected via ethernet cable
Would it matter if my installer setup RS485-1 as multi device and RS485-2 as Solaredge leader
Both HA and inverter are on same network
Any help you can offer would be greatly appreciated

### Expected behavior

connecting to modbus

### Diagnostic File

[home-assistant_solaredge_modbus_multi_2024-04-06T23-40-36.747Z.log](https://github.com/WillCodeForCats/solaredge-modbus-multi/files/14895241/home-assistant_solaredge_modbus_multi_2024-04-06T23-40-36.747Z.log)

### Home Assistant Version

2024.4.0

### solaredge-modbus-multi Version

current

### Installation Type

Core

### Read the Instructions

- [X] I have read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki
- [X] My configuration follows https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration
- [X] This issue is not about a template error (template errors should not be reported as an issue, read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Design-Notes)

**Comment by WillCodeForCats** [comment]
Read issue #573 and turn off "Auto-Detect Additional Entities".

**Reply by geos1200** [reply]
Appreciate the quick reply, you offer better support then Solar Edge.   If I turn on auto on anything it goes back to failing.

**Reply by geos1200** [reply]
So auto detect meters is now on and I have meters coming through, however if I add anything else it fails. And I don't seem to see the 2 lg batteries.

**Reply by WillCodeForCats** [reply]
It's an issue with the inverter firmware or config not responding to extra/battery related addresses. Some inverters allow it, others do not. The inverter is supposed to respond with an address error if you try to access things it doesn't support, but some versions don't respond. This is probably a SolarEdge bug.

There is discussion here about inverters supporting batteries: https://github.com/WillCodeForCats/solaredge-modbus-multi/discussions/63

I don't personally have batteries so unfortunately I can't help much with them.

**Reply by geos1200** [reply]
Thanks again, no issue, Appreciate all of the help.  Hate to continue to bother but where do you store the sensor information?  I am very new to HA, so I might be completely wrong.  The meters I do have don't all seem to be available to add to the dashboard objects, GUI states something about not having the right device class or state class etc.  I am trying to get the energy dashboard that was default configured with you sensors and some are there just not the ones I think I need.  
https://www.home-assistant.io/docs/energy/faq/#troubleshooting-missing-entities

**Reply by WillCodeForCats** [reply]
Energy dashboard is fully supported.

Use `sensor.solaredge_sim_i1_ac_energy` and `sensor.solaredge_sim_i2_ac_energy` for solar production.

Use `sensor.solaredge_sim_m2_exported` for energy to grid.

Use `sensor.solaredge_sim_m2_imported` for energy from grid.

From your log file M2 is the export+import meter.
